core/src/main/java/jenkins/fingerprints/FileFingerprintStorage.java

```diff
@@ -255,4 +317,26 @@ private static String serialize(Fingerprint.RangeSet src) {
         return buf.toString();
     }
 
+    /**
+     * Deletes a directory if it's empty.
+     */
+    private void deleteIfEmpty(File dir) {
+        String[] r = dir.list();
+        if(r==null)     return; // can happen in a rare occasion
+        if(r.length==0)
```

Comment:
curlies

Reply:
This is moved code, so will make this change in separate PR.